Content/Guides/syslog-ng-guide-admin/chapters/syslog-ng-whatsnew.htm

```diff
@@ -1,75 +1,34 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd">
     <head>
         <meta name="description" content="" />
     </head>
     <body name="syslog-ng_whatsnew">
         <h1 name="syslog-ng_whatsnew">What is new in <MadCap:variable name="General.product"></MadCap:variable> version <MadCap:variable name="Version.version"></MadCap:variable>?</h1>
         <p>This section lists the most recent changes of <MadCap:variable name="General.product" /> (<MadCap:variable name="General.OSE" />).</p>
-        <div>
-            <h6>Version 3.32 - 3.33</h6>
             <ul>
                 <li>
-                    <p><b>New destination: mqtt()</b>
+                    <p><b>New options for the kafka() destination C implementation</b>
                     </p>
-                    <p>From version 3.33, you can use the <span class="Code">mqtt()</span> destination to publish messages to MQTT brokers.</p>
-                </li>
+                    <p>Options <span class="Code">batch-lines()</span> and <span class="Code">batch-timeout()</span> have been added.</p>
+			</li>
+			<li>
+				<p><b>Proxy Protocol support</b>
+				</p>
+				<p>Proxy Protocol is now supported in the network() source.</p>
+				<p>If you connect load balancers to your <MadCap:variable name="General.OSE" /> application, <MadCap:variable name="General.OSE" /> identifies every connection that is connected to the load balancers identically by default, regardless of the source IP or the source protocol. Essentially, the load balancer masks the source IP unless you enable <a href="https://www.haproxy.com/blog/haproxy/proxy-protocol/">Proxy Protocol</a> support for your proxy TLS <span class="Code">transport()</span> to inject information about the original connection into the forwarded TCP session.</p>
+			</li>
+			<li>
+				<p><b>New --remove-orphans option in syslog-ng-ctl stats</b>
+				</p>
+				<p>New option <span class="Code">--remove-orphans</span> has been added to the <span class="Code">stats</span> command.</p>
+			</li>
+			<li>
+				<p><b>New monitoring() source driver</b>

```

Comment:
monitoring() source is a PE-only driver. Please remove, or add a conditional variable that ensures it will only be generated into a PE doc.